### PR TITLE
PDF error states

### PR DESCRIPTION
**Pull request by finben**
Addresses #228 I think it looks pretty good. The only question I have is should we be using the same error state for no url as when it doesn't load? On one hand it looks nice that the error state would be consistent. But if there is no url maybe our error state isn't about us saying "something went wrong" as much as it is about there just not being a url for that song because it doesn't have sheet music? 
 
Simulator Screen Shot - iPhone 8 - 2020-06-11 at 13 39 53

### CODE REVIEW

**Comment by skywalkerdude** [comment]
That's a good question. I think if the URL is empty, we probably just don't show the tab at all, right? I believe we already take care of that in DisplayHymnViewModel. Can you just change it to a non-optional and see if it works? I believe it should just work off the bat

File: Hymns/Common/PDFViewer.swift [hunk 1]
```diff
@@ -13,7 +13,7 @@ struct PDFViewer: UIViewRepresentable {
 
     func updateUIView(_ pdfView: PDFView, context: Context) {
         guard let url = url else {
-            // TODO create error state
+            displayErrorState(pdfView)
             return
```
Comment (skywalkerdude) [inline Hymns/Common/PDFViewer.swift:17]: I think url can just straight up be non-optional, and everything should still work

Comment (finben) [reply Hymns/Common/PDFViewer.swift:17]: The URL type is always an optional value. I don't even think it is possible to store a URL without an optional. Can I just do `guard let url = url else {
return } ` and leave it at that

**Comment by finben** [comment]
Lol... well you proved me wrong xD

**Pull request merged**